sh2lib/examples/http2_request/main/http2_request_example_main.c

```diff
@@ -167,5 +168,5 @@ void app_main(void)
      */
     ESP_ERROR_CHECK(example_connect());
 
-    xTaskCreate(&http2_task, "http2_task", (1024 * 32), NULL, 5, NULL);
+    xTaskCreate(&http2_task, "http2_task", CONFIG_EXAMPLE_NGHTTP_TASK_STACK_SIZE, NULL, 5, NULL);
```

Comment:
Should we have some compile time check to check that `EXAMPLE_NGHTTP_TASK_STACK_SIZE > ESP_NGHTTP_INBOUND_BUFFER_LENGTH` ? Maybe we can show a warning during the build?

Reply:
Added compile time check for this